vite-hardhat/README.md

```diff
@@ -49,11 +49,17 @@ Want to get started in a pinch? Start your project in a free Github Codespace!
 You can run the [example test file](./test/index.test.ts) with
 
 ```bash
-bun test
+bun run test
 ```
 
 This test shows the basic usage of Noir in a TypeScript Node.js environment. It also starts its own network and deploys the verifier contract.
 
+If you want to test only `UltraHonk`, you can run:
+
+```bash
+bun run test UltraHonk
```

Comment:
Seeing this error when I run this line. Other than that, looks good.

![Screenshot from 2024-12-19 12-25-56](https://github.com/user-attachments/assets/180cf72c-3ef0-4078-8b49-da34dbe852fc)

Reply:
fixed :)